src/system.h

```diff
@@ -210,7 +210,7 @@ struct System {
 
     void LoadOpRef() { LoadDB(frame->GetDocPath(L"examples/operation-reference.cts")); }
 
-    Cell *&InitDB(int sizex, int sizey = 0) {
+    unique_ptr<Cell> &InitDB(int sizex, int sizey = 0) {

```

Comment:
This is not great use of `unique_ptr`. `unique_ptr` is supposed to represent the owner. To borrow a `unique_ptr` just use a regular pointer.

A reference to a `unique_ptr` is funny.. it would maybe make sense if you were trying to modify the contents of the `unique_ptr` elsewhere but that doesn't seem to be the case here.